ptclient/cts/spi/dx/cl/ctMultiChangeDxa.vue

```diff
@@ -23,27 +23,22 @@
                 >
               </el-form-item>
             </el-form>
-            <DxaHistoryItem :dx="dx"></DxaHistoryItem>
+            <dxa-history-item :dx="dx" />
```

Comment:
Self-closing components where no slots or child nodes are relevant. Introduces kebab-case component names to match component names used elsewhere in the app. @vikaskedia do you feel this would help keep things consistent?

Reply:
There is also a lintung rule which can be set to enforce component name casing.